pipelines/package/package.yml

```
@@ -0,0 +1,10 @@
+parameters:
+  platform: ''
+
+steps:
+- template: ${{ format('{0}.yml', parameters.platform) }}
```

Isn't the the use of `format` unnecessary here and this can be simplified to this?
```suggestion
- template: ${{parameters.platform}}.yml
```